shapash/explainer/smart_explainer.py

```diff
@@ -233,8 +235,16 @@ def compile(self, x, model, explainer=None, contributions=None, y_pred=None,
                     )
                 else:
                     raise NotImplementedError('ACV does not support regression case yet.')
+            elif backend.lower() == 'lime':
+                contributions = lime_contributions(model=model,
+                                                   x_init=self.x_init,
+                                                   x_pred=self.x_pred,
+                                                   mode=self._case,
+                                                   num_classes=len(self._classes))
```

Comment:
It seems that the x_train parameter is missing here. 
It should be possible to access it thanks to the kwargs parameter of the compile method.